### DISCUSSION TITLE
Deploy Gollum on Netlify

### DISCUSSION THREAD

**Discussion by leobenkel** [gollum/gollum#1840] [General]
Hello, 

I thought you might be interested to know that I am trying to deploy Gollum on Netlify (https://answers.netlify.com/t/deploy-gollum-on-netlify/72644). I will keep you up to date if I get help there or I would also appreciate it if you have any tips to provide :) 

Thank you !

**Comment by benjaminwil** [comment]
That's great. I honestly didn't know Netlify worked for non-static sites–I would have thought this was impossible. But after skimming their features page it looks like maybe non-static sites can be deployed now, too.

Since no code changes should be required in Gollum for deployment, I'm going to convert this issue to a discussion thread.

Wish I had tips for you, but I don't. Good luck!

**Comment by leobenkel** [comment]
thanks, i'll let you know if it is possible 🤞
